common/rfb/SConnection.h

```diff
@@ -192,6 +192,10 @@ namespace rfb {
     // from queryConnection() or some time later.
     virtual void queryConnection(const char* userName);
 
+    // desktopReady() is called when the desktop has finished
+    // initializing and is ready for a client.
+    virtual void desktopReady();
+
```

Comment:
This is in the wrong section. This should be in the interface part as it is to be called by another object.